core/src/main/java/com/linecorp/armeria/server/streaming/JsonTextSequences.java

```diff
@@ -322,5 +374,17 @@ private static HttpData toHttpData(ObjectMapper mapper, @Nullable Object value)
         }
     }
 
+    private static <T> HttpData toHttpData(Function<? super T, String> contentConverter, @Nullable T value) {
+        final String content = contentConverter.apply(value);
+        requireNonNull(content, "contentConverter.apply() returned null");
+        final byte[] contentBytes = content.getBytes(StandardCharsets.UTF_8);
```

Comment:
I think we still convert the content string into a JSON string. For example, if `content.equals("foo")`, it has to be encoded into `"foo"`, not `foo`. Similarly, when `content.equals("\"")`, it has to be encoded into `"\""`, not `"`.